tests/components/group/test_config_flow.py

```diff
@@ -484,20 +484,20 @@ async def test_options_flow_hides_members(
 ]
 LOCK_ATTRS = [{"supported_features": 1}, {}]
 MEDIA_PLAYER_ATTRS = [{"supported_features": 0}, {}]
-SENSOR_ATTRS = [{"icon": "mdi:calculator"}, {"max_entity_id": "sensor.input_two"}]
+SENSOR_ATTRS = [{"icon": "mdi:calculator"}, {"max_entity_id": "sensor.input_one"}]
 
 
 @pytest.mark.parametrize(
     ("domain", "extra_user_input", "input_states", "group_state", "extra_attributes"),
     [
-        ("binary_sensor", {"all": True}, ["on", "off"], "off", [{}, {}]),
+        ("binary_sensor", {"all": True}, ["on", "off"], "on", [{}, {}]),
```

Comment:
This one will see the first state change in time now.  Ideally we add another test in the future to check for both state changes.